index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebDev Summit </title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+        <!-- Navigation Section -->
+        <header>
+            <div class="contact-us-navbar">
+                <ul class="contact-us-navbar-ul">
+                    <li class="fa fa-facebook contact-us-navbar-li"></li>
+                    <li class="fa fa-twitter contact-us-navbar-li"></li>
+                    <li class="contact-us-navbar-li">English</li>
+                    <li class="contact-us-navbar-li">My page</li>
+                    <li class="contact-us-navbar-li">Logout</li>
+                </ul>
+            </div>
+            <nav>
+                <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+                <div class="logo">
+                    <a href="./index.html"><img src="./assets/imgs/logo.png" alt="Logo"></a>
+                </div>
+                <div class="menu">
+                    <ul class="menu-list-ul">
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <!-- Main Content Section -->
+        <main>
+        <section class="hero-section">
+            <div class="hero-div">
+                <h1 class="thin-text">"Hello! Sharing Knowledge around the Globe"
+                    <h2 class="thick-text">Awareness Summit about web development 2023</h2>
+                </h1>
+                <span class="description-about-summit">
+                     A joyful and informative enviroment for sharing creative ideas about the advancements of technology.
+                </span>
```

Comment:
[OPTIONAL]
- [ ] Kindly note the `width` of the container of the description of the event on the Figma design, It is different on your design. Please check it to follow the project requirement
### Your design :point_down: 
![Screenshot from 2023-06-27 15-06-15](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/91142432/46c07922-e0b6-460c-815b-8e39f6597198)
### Figma design :point_down: 
![Screenshot from 2023-06-27 15-09-50](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/91142432/6762c683-dc0f-42e3-bc60-cda2cd78d699)

**This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change**